x-pack/plugins/reporting/public/components/job_warning_formulas.tsx

```diff
@@ -21,7 +22,7 @@ export const getWarningFormulasToast = (
     <FormattedMessage
       id="xpack.reporting.publicNotifier.csvContainsFormulas.formulaReportTitle"
       defaultMessage="Report may contain formulas {reportObjectType} '{reportObjectTitle}'"
-      values={{ reportObjectType: job.type, reportObjectTitle: job.title }}
+      values={{ reportObjectType: job.jobtype, reportObjectTitle: job.title }}
```

Comment:
Had to rename this because it is confusing: the function that creates a `JobSummary` object was renaming the field name of `ReportSource` from `jobtype` so now they're the same.